docs/graphkb/matching/gene.md

```diff
@@ -0,0 +1,45 @@
+# Gene Example
+
+The [`get_equivalent_features()`](./../reference/graphkb/match/#get_equivalent_features) method is
+used to find genes *equivalent* to the input/target feature.
+
+```python
+from graphkb.match import get_equivalent_features
+
+genes = get_equivalent_features(graphkb_conn, 'KRAS')
+```
+
+This will use a similar algorithm to what we have seen above in the disease matching example.
+
+![initial graph](../images/pori-gene-matching-1.png)
+
+In the graph above the relationship types shown are: `GeneralizationOf` (G), `ElementOf` (E),
+`DeprecatedBy` (D), and `CrossReferenceOf` (X).
+
+## Match by Name
+
+Like before, the first thing done is to match the input name
```

Comment:
would possibly use 'As' instead of 'Like'

Reply:
ya that sounds better. changed!